test/index.ts

```diff
@@ -10,14 +10,12 @@
 // to report the results back to the caller. When the tests are finished, return
 // a possible error to the callback or null if none.
 
-import { Globals } from '../src/globals';
-
-var testRunner = require('vscode/lib/testrunner');
-
+import Globals from '../src/globals';
 Globals.isTesting = true;
 
 // You can directly control Mocha options by uncommenting the following lines
```

Comment:
maybe delete this first sentence in this comment now? Not sure it makes much sense, the 2nd sentence is still useful